### PR TITLE
Fix for not finding version as key when it is integer

### PR DESCRIPTION
**Pull request by thanajirao**

### CODE REVIEW

File: ad-insertion/video-analytics-service/app/modules/PipelineManager.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def stop_instance(name, version, instance_id):
     @staticmethod
     def is_pipeline_exists(name, version, instance_id=None):
         if name not in PipelineManager.pipelines or \
-                version not in PipelineManager.pipelines[name]:
+                str(version) not in PipelineManager.pipelines[name]:
```
Comment (nnshah1) [inline ad-insertion/video-analytics-service/app/modules/PipelineManager.py:138]: can we store version in map as integer?

Comment (thanajirao) [reply ad-insertion/video-analytics-service/app/modules/PipelineManager.py:138]: I think we should but I just found this issue, strangely it works sometime and doesn't at some places

Comment (thanajirao) [reply ad-insertion/video-analytics-service/app/modules/PipelineManager.py:138]: I mean integer as key works at some places in code without any issues

Comment (nnshah1) [reply ad-insertion/video-analytics-service/app/modules/PipelineManager.py:138]: let's put a task / item to clean that up - need to make it consistent

**Pull request merged**